tests/wikidata/test_extract.py

```diff
@@ -156,6 +164,47 @@ def test_search_organization_by_label_mocked(monkeypatch: MonkeyPatch) -> None:
         },
     }
 
+    expected_query_response = [
+        {
+            "item": {"type": "uri", "value": "http://www.wikidata.org/entity/Q26678"},
+        },
+    ]
+
+    def mocked_query_response():
+        return expected_query_response
+
+    monkeypatch.setattr(
+        WikidataQueryServiceConnector,
+        "get_data_by_query",
+        lambda self, _: mocked_query_response(),
+    )
+
+    def mocked_item_details_response():
+        with open(TESTDATA_DIR / "items_details.json", encoding="utf-8") as f:
+            data = json.load(f)
+            return data[0]
+
+    monkeypatch.setattr(
+        WikidataAPIConnector,
+        "get_wikidata_item_details_by_id",
+        lambda self, _: mocked_item_details_response(),
+    )
+
+    search_results = search_organization_by_label(item_label="TEST")
+
+    if search_results:
+        assert search_results.model_dump() == expected_item_details_response
+    else:
+        pytest.fail(
```

Comment:
see above

Reply:
updated in 108989566acf52f31953594e8d8cbd61be22be7a